docs/go-error-handling.md

```diff
@@ -3,12 +3,19 @@ id: go-error-handling
 title: Error Handling
 ---
 
-An activity, or child workflow, might fail, and you could handle errors differently based on different
-error cases. If the activity returns an error as `errors.New()` or `fmt.Errorf()`, those errors will
-be converted to `*temporal.ApplicationError` and wrapped inside `*temporal.ActivityTaskError` or `*temporal.ChildWorkflowExecutionError`. If the activity returns an error as
-`temporal.NewRetryableApplicationError("error message", details)`, that error will be returned as `*temporal.ApplicationError`.
+An activity, or a child workflow, might fail, and you could handle errors differently based on the different
```

Comment:
Capitalize core terms when they are used as such. I.E:
activity -> Activity
workflow -> Workflow
See glossary for core terms: https://docs.temporal.io/docs/glossary